externs/olx.js

```diff
@@ -4975,14 +4974,6 @@ olx.source.ImageStaticOptions.prototype.imageExtent;
 
 
 /**
- * Size of the image in pixels.
- * @type {ol.Size|undefined}
- * @api stable
- */
-olx.source.ImageStaticOptions.prototype.imageSize;
```

Comment:
@ahocevar your change looks like a definite improvement.  We should clarify exactly where the size property is needed.  I'm only guessing about the SVG/IE case because of a recent experience with an icon symbolizer.